docs/tutorials/configure/pet-photographer.md

```diff
@@ -0,0 +1,984 @@
+
+- To choose the camera the robot will use, click the **video path** field.
+  - If your robot is connected, you'll see a list of available cameras.
+  - Select the camera you want to use, then click **Save config**.
+
+![An instance of the webcam component named 'cam'](/tutorials/pet-photographer/webcam-component.png)
+
+### Add colorfilter component
+
+1. Click the **Components** subtab and then click **Create component**.
+1. Next, select the `local modular resource` type from the list.
+   {{<imgproc src="/tutorials/pet-photographer/add-local-module-select.png" resize="300x" declaredimensions=true alt="The add a component modal showing the list of components to add with 'local modular resource' shown at the bottom">}}
+1. On the following screen:
+
+   1. Select the camera from the drop down menu.
+   1. Enter the {{< glossary_tooltip term_id="model-namespace-triplet" text="model namespace triplet">}} for your modular resource's {{< glossary_tooltip term_id="model" text="model" >}}, `example:camera:colorfilter`.
+   1. Provide a name for this instance of your modular resource.
+      This name must be different from the module name.
+
+      {{<imgproc src="/tutorials/pet-photographer/add-colorfilter-module-create.png" resize="400x" declaredimensions=true alt="The add a component model showing the create a module step for a local color filter module">}}
+
+1. Click **Create** to create the modular resource component.
+1. Copy the following JSON configuration into the **Attributes** section:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+     "vision_service": "my_color_detector",
+     "actual_cam": "cam"
+   }
+   ```
+
+   ![A component panel for a color filter modular resource with the attributes filled out for vision service and actual_cam](/tutorials/pet-photographer/colorfiltercam-component-attributes.png)
+
+### Configure data capture
+
+To add data capture for the color filter camera, click **Add Method** in the **Data Capture configuration** section of your color filter camera component.
+Toggle the **Type** dropdown menu, select **ReadImage**, and set the **Frequency** of the capture to `0.1`.
+Then, click **Save config**.
+
+![A component panel for a color filter modular resource with the attributes filled out for vision service and actual_cam as well as the data capture configuration capture set capture ReadImage at 0.1 frequency](/tutorials/pet-photographer/colorfiltercam-component.png)
+
+## Test your color filter camera
+
+To test that your color filter camera is capturing and filtering images properly, navigate to the **Control** tab on your robot's page.
+
+On the **colorfiltercam** panel, toggle **view colorfiltercam** to view your camera's live feed.
+Test the filter by moving a blue colored item within the camera's field of view.
+Then, go to the **Data** tab to view pictures that contain the blue colored item.
+
+For example, your pet with the blue collar:
```

Comment:
It's not abundantly clear from the image itself, but the ah-ha moment here is that this image _only shows images that contain the object, all other images (without that object) were never captured in the first place._

What about the following tweak to try to bring that home:

```suggestion
For example, the following is the results of several dozen pictures of the same dog, but only those pictures where he is wearing the blue collar were captured and synced to the cloud:
```